docs/manual/constant-value-checker.tex

```diff
@@ -57,10 +57,11 @@ \subsection{Type Annotations\label{constant-value-checker-type-annotations}}
 could be more than 10 different values, the Constant Value
 Checker gives up and its type becomes
 \refqualclass{common/value/qual}{IntRange} for integral types,
-\refqualclass{common/value/qual}{ArrayLenRange} for array types, and
+\refqualclass{common/value/qual}{ArrayLenRange} for array types,
```

Comment:
It would be helpful to explain here how `StringVal` is actually converted into `ArrayLen` or `ArrayLenRange` when there are too many values.